src/groups.ts

```diff
@@ -91,6 +100,11 @@ export const selectGroupsForSelectedElements = (
     }
   }
 
+  nextAppState.selectedElementIds = makeNextSelectedElementIds(
+    nextAppState.selectedElementIds,
```

Comment:
Does this actually improve anything? This function is being called on every mouse down, so each time `makeNextSelectedElementIds` would be recalculated.  I've rewritten this whole function due to `selectGroup` being a bottleneck, IMO this would be better off with caching (which I already have). Would prefer to leave it for a separate PR.

Reply:
irrespective of being cached or not, we want the cache-miss to not update the `selectedElementIds` unnecessarily. Generally, whenever we update the `selectedElementIds`, we should use this helper, without giving it much thought. It shouldn't hurt.